pkg/storage/trb_request_feedback.go

```diff
@@ -123,28 +153,13 @@ func (s *Store) GetTRBRequestFeedbackByTRBRequestID(ctx context.Context, trbRequ
 // matching the given TRB request ID
 func (s *Store) GetNewestTRBRequestFeedbackByTRBRequestID(ctx context.Context, trbRequestID uuid.UUID) (*models.TRBRequestFeedback, error) {
 	feedback := models.TRBRequestFeedback{}
-	stmt, err := s.db.PrepareNamed(`
-		SELECT *
-		FROM trb_request_feedback
-		WHERE trb_request_id = :trb_request_id
-		ORDER BY created_at DESC LIMIT 1`)
-	if err != nil {
-		appcontext.ZLogger(ctx).Error(
-			"Failed to fetch latest TRB request feedback",
-			zap.Error(err),
-			zap.String("trbRequestID", trbRequestID.String()),
-		)
-		return nil, err
-	}
-	defer stmt.Close()
-
-	arg := map[string]interface{}{"trb_request_id": trbRequestID}
-	err = stmt.Get(&feedback, arg)
+	err := namedGet(ctx, s, &feedback, sqlqueries.TRBRequestFeedback.GetByNewestByID, args{
+		"trb_request_id": trbRequestID,
+	})
 
-	hasFeedback := true
 	if err != nil {
 		if errors.Is(err, sql.ErrNoRows) {
-			hasFeedback = false
+			return nil, nil
```

Comment:
nit because the below line isn't your code, but now that this returns, you can remove `else` and remove some nesting/indentation

Reply:
I'll make this change in the next one, thanks!